Inference/Image_Classifier.py

```diff
@@ -23,10 +23,15 @@
 # 2. Pre-processing: create npy files
 
 # Load pre-processed dataset. It should have been pre-processed (tissue type identification) first.
+print('Loading file parameters...', end='')
 tile_dataset = LoadFileParameter(config, SVS_dataset)
+tile_dataset_full = tile_dataset.copy()  # keep the full tile_dataset for final saving, but only process the reduced.
+valid_tumour_tiles_index = tile_dataset_full['prob_tissue_type_Tumour'] > 0.94
+tile_dataset = tile_dataset.loc[valid_tumour_tiles_index]
 
-# Mask the tile_dataset to only keep the tumour tiles, depending on a pre-set criteria.
-tile_dataset = tile_dataset[tile_dataset['prob_tissue_type_tumour'] > 0.85]
+# Assign SVS paths based on SVS_dataset (this will change depending on which workstation is running the script).
+tile_dataset['SVS_PATH'] = tile_dataset['id_external'].map(dict(zip(SVS_dataset.id_external, SVS_dataset.SVS_PATH)))
```

Comment:
Unclear what you are doing in here, isnt SVS_Path part of the Data Path in the Config?

Reply:
Moved to LoadFileParameter